src/modules/delete.js

```diff
@@ -11,6 +10,9 @@ export default function deleteTask() {
       container.removeChild(item);
       const deletedTask = e.target.previousElementSibling.textContent;
       tasks = tasks.filter((task) => task.description !== deletedTask);
+      tasks.forEach((task, i) => {
+        task.index = i;
+      });
       localStorage.setItem('tasks', JSON.stringify(tasks));
```

Comment:
By default, new tasks should have the property index set to the value of the new array length (i.e., if a 5th task is added to the list, the index of that task should equal 5), and deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. 😉

For example, if I have to-dos with an index of 1-5 and I delete the 3rd to-do task, I should have an index of 1-4, not 0-3 or 2-5 👍🏽